doc/release/next_whats_new/pie_label.rst

```diff
@@ -0,0 +1,28 @@
+Adding labels to pie chart wedges
+---------------------------------
+
+The new `~.Axes.pie_label` method adds a label to each wedge in a pie chart created with
+`~.Axes.pie`.  It can take
+
+* a list of strings, similar to the existing *labels* parameter of `~.Axes.pie`
+* a format string in analogy to the existing *autopct* parameter of `~.Axes.pie` except
+  that it uses the `str.format` method, and it can handle absolute values as well as
+  fractions/percentages
```

Comment:
```suggestion
* a format string similar to the existing *autopct* parameter of `~.Axes.pie` except
  that it uses the `str.format` method and it can handle absolute values as well as
  fractions/percentages
```

Reply:
done